src/Interpreters/InterpreterSelectQuery.cpp

```diff
@@ -2122,7 +2121,7 @@ void InterpreterSelectQuery::applyFiltersToPrewhereInAnalysis(ExpressionAnalysis
     else
     {
         /// Add row level security actions to prewhere.
-        analysis.prewhere_info->row_level_filter = std::move(analysis.filter_info->actions);
+        analysis.prewhere_info->row_level_filter = analysis.filter_info->actions;
```

Comment:
What about next line? Should we also remove the std::move of `analysis.filter_info->column_name`